python/TestHarness/tests/test_JSONDiff.py

```diff
@@ -1,26 +0,0 @@
-#* This file is part of the MOOSE framework
-#* https://www.mooseframework.org
-#*
-#* All rights reserved, see COPYRIGHT for full restrictions
-#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
-#*
-#* Licensed under LGPL 2.1, please see LICENSE for details
-#* https://www.gnu.org/licenses/lgpl-2.1.html
-
-import subprocess
-from TestHarnessTestCase import TestHarnessTestCase
-
-class TestHarnessTester(TestHarnessTestCase):
```

Comment:
We should not remove this

Reply:
These tests are completely encapsulated by the new schemadiff tests, which run JSONDiff tests that do exactly this and more

Reply:
They're different - this one is more exhaustive and there's no reason to remove it as it should still work, spare for maybe a few changes

Reply:
The actual "jsondiff" tests that this python runs were directly renamed and brought into the schemadiff tests, with more functionality added. The new version encompasses all of these, adds more, and I stand by my statement